src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx

```diff
@@ -14,8 +14,9 @@ import {
 } from 'DetailsView/components/save-assessment-button';
 import {
     LoadAssessmentButton,
+    LoadAssessmentButtonDeps,
     LoadAssessmentButtonProps,
-} from 'DetailsView/components/load-assessment-button';
+} from '../../../../../DetailsView/components/load-assessment-button';
```

Comment:
Was there any particular reason for making this import path relative?

Reply:
Sometimes the paths give me an error and it recommends this as the fix. Will switch back and make sure to run the build when switching branches!